src/components/Chips/__stories__/chips.stories.mdx

```diff
@@ -149,6 +149,40 @@ e.g. `<Chips label="GRASS_GREEN" color={Chips.colors.GRASS_GREEN} />`
   </Story>
 </Canvas>
 
+### Overflowing chip
+
+[Demonstration examples]
+
```

Comment:
Is it according to design review?

Reply:
It's just an "Demonstration examples" :) Probably don't want to show it at all in storybook - I've asked Meirav about that, waiting for an answer

Reply:
Removed it for now